### DISCUSSION TITLE
Please, make it possible to mount the glowstick on the head like a flashlight.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#11268] [Ideas and suggestions]
It is very strange to me, that the glowstick only shines when you hold it in your hand.
I think that it is absolutely necessary to make it possible to attach glowstick to the head instead of a flashlight.
It would be ideal to add a pocket for glowstick/medscaner/flashlight/thermal goggles
in all helmets.
In reality, you can easily attach the glowstick to your body, with your hands remain free.
Now, in the game, you have to throw glowstick, every time when you need to use, for example, a cutter. And then pick glowstick up again.

**Comment by mygamingaccount** [comment]
Sounds like an easy thing to mod!

**Comment by BiffW** [comment]
It would be also cool if you can hook a glowing stick on your toolbelt.
